plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/security/KafkaSecurityModule.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.security;
+
+import com.google.inject.Binder;
+import com.google.inject.Module;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+import io.trino.plugin.kafka.KafkaConfig;
+
+import static io.airlift.configuration.ConditionalModule.installModuleIf;
+import static io.airlift.configuration.ConfigurationModule.installModules;
+
+public class KafkaSecurityModule
```

Comment:
Shouldn't this be part of `DEFAULT_EXTENSION`? Isn't it the case that if one overrides the clients probably also want to override this one?

Reply:
Got it. I'll add it under `DEFAULT_EXTENSION` as well.